radio/src/gui/480x272/lcd.h

```diff
@@ -133,7 +133,7 @@ void lcdDrawNumber(coord_t x, coord_t y, int32_t val, LcdFlags flags=0, uint8_t
 
 void putsModelName(coord_t x, coord_t y, char *name, uint8_t id, LcdFlags att);
 void putsStickName(coord_t x, coord_t y, uint8_t idx, LcdFlags att=0);
-void drawSwitch(coord_t x, coord_t y, swsrc_t swtch, LcdFlags flags=0);
+void drawSwitch(coord_t x, coord_t y, int32_t swtch, LcdFlags flags=0);
```

Comment:
And you (or someone) may remember it again if the definition changes... :)

I tried a few things... the error in lcd.h happens because layout.h is included from datastructs.h.  So one way around it might be to separate LayoutFactory into its own file and include lcd.h from there.

Reverted for now and "fixed" `mixsrc_t`.